.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 The Ground Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: End to End Test
+
+on:
+  issue_comment:
+    types: [created]
+  workflow_call:
+
+jobs:
+  e2eTest:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/e2eTest')
+    steps:
+      - name: Start test
+        run: |
+          echo "Begin end to end test"
+          
+
+  createTest:
+    needs: e2eTest
+    name: Create a new survey
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run create-test
+        uses: ./.github/actions/create-test
+        with:
+          platform-repository: ${{ github.repository }}
+          upload-artifacts: true
+
+  submitTest:
+    needs: createTest
+    name: Submit to survey
```

Comment:
Nit: "Submit data to survey", or even better "Collect and submit data"

Reply:
Nice! Done.